tools/ci_build/amd_hipify.py

```diff
@@ -286,10 +276,13 @@
~        "#include <hipcub/backend/rocprim/iterator/discard_output_iterator.hpp>",
~    )
~    s = s.replace("#include <cub/util_allocator.cuh>", "#include <hipcub/util_allocator.hpp>")
     s = s.replace('#include "cub/util_allocator.cuh"', "#include <hipcub/util_allocator.hpp>")
```

Comment:
maybe install "black" and see why it's complaining

Reply:
I didn't get it. What is the "black".
Btw the indents actually align well and I'm not sure GitHub can't display correctly.

Reply:
black is the python format tool used in the pipeline - pip install black
https://pypi.org/project/black/

Reply:
Thank you!
It did change something, although I'm not sure what it changed